### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by j-hellenberg**
We need to figure out what we actually want to write about in the README. Right now, I've kept it rather concise as I felt that looking at the `KVAC` class and the corresponding tests should already be a pretty good starting point. I don't really think we need to duplicate the usage example to the README if you can just go to these files and look it up

### CODE REVIEW

**Comment by leonschmidt99** [comment]
Imo the usage example should be part of the README instead of this code comment in `kvac.py`. It should document:
- how you are supposed to use the kvac library
- what the major functions do automatically (verify NIZKs and all that jazz)
- the limitations, e.g. that messages must be 16 bytes should be clearly stated

I'll see if I can get around to adding this soon.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+was dropped in favour of implementing a more generic version of the KVACs (as some implementation details of libsignal 
+prevented us from doing so).
+
+THE AUTHORS DO NOT ASSUME RESPONSIBILITY FOR THE CORRECTNESS OF THE PERFORMED CRYPTOGRAPHIC OPERATIONS.
+DO NOT USE THIS PROJECT IN A PRODUCTION ENVIRONMENT.
+
+In addition to the unclear functional cryptographic security of this implementation, the authors expect side channel 
+attacks to be possible due to use of the Python programming language without any precautions taken to prevent these 
+types of attacks.
+
```
Comment (konradh) [inline README.md:22]:
```suggestion
```

I would just drop it as it is speculation. The text above is enough.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+This module was developed as part of the 2022/23 *Current Topics in Group Messaging* seminar at the
+[Hasso-Plattner-Institute (HPI)](https://hpi.de) trying to reproduce the Signal Private Group system described
+by the paper ["The Signal Private Group System and Anonymous Credentials Supporting Efficient Verifiable Encryption" by
+Chase et al.](https://eprint.iacr.org/2019/1416.pdf) Therefore, it is supposed to solely fulfill academic purposes.
+Note that the original goal of reaching full compatibility with the [libsignal Rust implementation](https://github.com/signalapp/libsignal)
+was dropped in favour of implementing a more generic version of the KVACs (as some implementation details of libsignal 
+prevented us from doing so).
+
+THE AUTHORS DO NOT ASSUME RESPONSIBILITY FOR THE CORRECTNESS OF THE PERFORMED CRYPTOGRAPHIC OPERATIONS.
+DO NOT USE THIS PROJECT IN A PRODUCTION ENVIRONMENT.
```
Comment (konradh) [inline README.md:17]:
```suggestion
**THE AUTHORS DO NOT ASSUME RESPONSIBILITY FOR THE CORRECTNESS OF THE PERFORMED CRYPTOGRAPHIC OPERATIONS.
THERE WAS NO REVIEW PERFORMED BY AN EXPERT.
DO NOT USE THIS PROJECT IN A PRODUCTION ENVIRONMENT.**
```

File: kvac/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.0.5"
+__version__ = "1.0.0"
```
Comment (konradh) [inline kvac/__init__.py:1]:
```suggestion
__version__ = "0.1.0"
```

1.0.0 suggests that the library is mature.

**Review by konradh — COMMENTED** [review]
Please make the two changes from https://github.com/hpicrypto/poksho/pull/19 here too.

**Pull request merged**